lighthouse-ci/mock-pages/fast-ads.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<!--
+* Copyright 2020 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+-->
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Fast Ads</title>
+    <script>googletag = window.googletag || {cmd: []};</script>
```

Comment:
Is there a reason this is defined before gpt.js is included? Normally our examples put it at the top of the script block immediately following the gpt.js include.